spec/integration/govuk_index/publishing_event_processor_spec.rb

```diff
@@ -1,99 +1,137 @@
 require 'spec_helper'
 
 RSpec.describe 'GovukIndex::PublishingEventProcessorTest' do
-  before do
-    bunny_mock = BunnyMock.new
-    @channel = bunny_mock.start.channel
-
-    consumer = GovukMessageQueueConsumer::Consumer.new(
-      queue_name: "bigwig.test",
-      processor: GovukIndex::PublishingEventProcessor.new,
-      rabbitmq_connection: bunny_mock
-    )
-
-    @queue = @channel.queue("bigwig.test")
-    consumer.run
+  context "Testing via fake rabbit queue" do
+    before do
+      bunny_mock = BunnyMock.new
+      @channel = bunny_mock.start.channel
+
+      consumer = GovukMessageQueueConsumer::Consumer.new(
+        queue_name: "bigwig.test",
+        processor: GovukIndex::PublishingEventProcessor.new,
+        rabbitmq_connection: bunny_mock
+      )
+
+      @queue = @channel.queue("bigwig.test")
+      consumer.run
+    end
+
+    it "should_save_new_document_to_elasticsearch" do
```

Comment:
can you make these strings into sentences?

Reply:
done